modules/vpc/user_management.yml.tmpl

```diff
@@ -0,0 +1,49 @@
+---
+Consul:
+  Server: 127.0.0.1:8500
+  Namespace: "environments/${environment}/global/nubis-users"
+
+AWS:
+  Region: "${region}"
+  SMTPFromAddress: "${smtp_from_address}"
+  SMTPUsername: "${smtp_username}"
+  SMTPPassword: "${smtp_password}"
+  SMTPHostname: "${smtp_host}"
+  SMTPPort:     "${smtp_port}"
+  AWSIgnoreUserList:
+    - nubis-bootstrap
+    - datadog
+    - gozer
```

Comment:
Makes me wonder, actually, on the wiping behaviour. If we have an app deployment that creates an IAM user to have it's own credentials to do something, what will happen?

Bugzilla, for instances, creates an IAM user with credentials so that the sync script in the DC can have working credentials to sync into a S3 bucket.